staging/src/k8s.io/cluster-bootstrap/util/secrets/secrets.go

```diff
@@ -46,21 +46,30 @@ func GetData(secret *v1.Secret, key string) string {
 
 // HasExpired will identify whether the secret expires
 func HasExpired(secret *v1.Secret, currentTime time.Time) bool {
+	_, expired := GetExpiration(secret, currentTime)
+
+	return expired
+}
+
+// GetExpiration get expiration time from now
```

Comment:
need to doc the following things:
* the meaning of the boolean (isExpired)
* the meaning of a zero-value duration when isExpired=false (no expiration)